Cargo.toml

```diff
@@ -39,6 +50,6 @@ default-features = false
 optional = true
 
 [dependencies.multipart]
```

Comment:
Sure, I didn't want to have to mess with the synchronous code too much while focusing on async is all. That's also why it's on hyper 0.10, as 0.11 introduced a bunch of api changes apparently while 0.10 was compatible with everything else I needed while still mostly compatible with the existing `Discord` impls.